pkg/controller/admissionchecks/provisioning/controller.go

```diff
@@ -313,7 +337,8 @@ func (c *Controller) syncOwnedProvisionRequest(
 			c.record.Eventf(wl, corev1.EventTypeNormal, "ProvisioningRequestCreated", "Created ProvisioningRequest: %q", req.Name)
 			activeOrLastPRForChecks[checkName] = req
 		}
-		if err := c.syncProvisionRequestsPodTemplates(ctx, wl, requestName, prc); err != nil {
+
```

Comment:
nit, revert empty line changes since this is to be cherry-picked, so to minimize changes of unnecessary conflicts.